packages/discord.js/typings/index.d.ts

```diff
@@ -5311,6 +5311,7 @@ export interface ClientOptions {
   ws?: WebSocketOptions;
   rest?: Partial<RESTOptions>;
   jsonTransformer?: (obj: unknown) => unknown;
+  automatedMessageNonce?: boolean;
```

Comment:
This is missing in the documentation.

Also, I think it would be best to name this `enforceNonce`, much like how `failIfNotExists` matches the actual property name.

Reply:
Updated